aerofiles/seeyou/reader.py

```diff
@@ -193,3 +193,90 @@ def decode_description(self, description):
             return None
 
         return description
+
+    def decode_taskname(self, fields):
+        if not fields:
+            raise ParserError('Name field must not be empty')
+
+        return fields[0]
+
+    def decode_tasklist(self, fields, waypoints):
```

Comment:
the best way to do this is creating a new branch at the same commit as my `master` branch (423ec3a4b6). after that your changes are still alive in your `master` branch (which corresponds to this PR) but not in that new branch (call it e.g. `seeyou-tasks`). in that new branch you should only focus on this one topic (adding tasks to the seeyou reader) and it would probably be best if you did one commit per checklist item above.

if you have any specific questions on how to do stuff with git let me know. otherwise I'd suggest working through some git tutorial to get a rough understanding of how to work with it. https://guides.github.com/activities/hello-world/ seems to be a good one for example.